jetson/nav/.clang-format

```diff
@@ -0,0 +1,66 @@
+# Generated from CLion C/C++ Code Style settings
```

Comment:
interesting, is this only possible w CMake not Meson?

Reply:
No, this is for IDEs. Both VSCode, CLion, VS, etc. support it. It is just a code style guide. Ideally we could lint with it in the future.

Reply:
As far as cmake / meson goes, Meson at least will notice you have this file and create a target for you to use it: https://mesonbuild.com/Code-formatting.html#clangformat

Reply:
Interesting.

Also... out of curiosity... how did you even find this to comment on. Lol

Reply:
@eli-schwartz have a read! https://www.rojtberg.net/1481/do-not-use-meson/